pkg/cmd/monitor_command/run_monitor_command.go

```diff
@@ -66,6 +67,13 @@ func NewRunMonitorCommand(ioStreams genericclioptions.IOStreams) *cobra.Command
 	cmd.Flags().StringVar(&monitorOpt.ArtifactDir,
 		"artifact-dir", monitorOpt.ArtifactDir,
 		"The directory where monitor events will be stored.")
+	cmd.Flags().StringVar(&monitorOpt.LoadBalancerType,
+		"lb-type", monitorOpt.LoadBalancerType,
+		"Set load balancer type, available options: internal-lb, service-network, external-lb (default)")
+	cmd.Flags().StringVar(&monitorOpt.ExtraMessage,
```

Comment:
the monitor is not an API disruption command. it is a command to look at all "important" happenings in a cluster and track them.

Reply:
Correct, but monitor is responsible for saving events as json - and this is where we'll need a way of adding more information (i.e. node name which originated the interval)